src/dataServices/dynamoDbDataService.ts

```diff
@@ -139,19 +145,88 @@ export class DynamoDbDataService implements Persistence {
         };
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     async initiateExport(initiateExportRequest: InitiateExportRequest): Promise<string> {
-        throw new Error('method not implemented');
+        await this.throttleExportRequestsIfNeeded(initiateExportRequest.requesterUserId);
+        // Create new export job
+        const jobId = uuidv4();
+
+        // TODO: Start Export Job Step Function
+        // const stepFunctionArn = await StartStepFunctionAndGetStepFunctionArn
+
+        const params = DynamoDbParamBuilder.buildPutCreateExportRequest(jobId, initiateExportRequest, '');
+        await this.dynamoDb.putItem(params).promise();
+        return jobId;
+    }
+
+    async throttleExportRequestsIfNeeded(requesterUserId: string) {
+        const MAXIMUM_SYSTEM_LEVEL_CONCURRENT_REQUESTS = 2;
+        const MAXIMUM_CONCURRENT_REQUEST_PER_USER = 1;

```

Comment:
maybe make them class constants instead of being inside the method?

Reply:
updated